Automattic/wp-calypso
# Earn: Update illustration and top banner styles

#### Changes proposed in this Pull Request
Creating more consistency by updating the styles of the top banner on the Earn section to be more in line with what we've established with My Home.

* Update the illustration to fit with the new style.
* Update the top banner styles and typography.

The overall height of the banner is reduced because an intro/welcome message doesn't need to be a huge element of the page like it currently is. These styles deviate slightly from the task card styles on My Home with extra padding to help it feel less dense.

Before | After
------------ | -------------
<img width="1089" alt="image" src="https://user-images.githubusercontent.com/448298/83187893-e0af2980-a0fc-11ea-8078-cffe372cbbb6.png"> | <img width="1098" alt="Screen Shot 2020-05-28 at 3 50 05 PM" src="https://user-images.githubusercontent.com/448298/83187656-7dbd9280-a0fc-11ea-87a5-d78e59e97351.png">


#### Testing instructions

* Check out this branch or visit the calypso.live link
* Visit the Earn section
* Confirm the banner has the new styles

Fixes part of #42371
